lib/Service/ActivityService.php

```diff
@@ -355,7 +355,7 @@ private function getMessageFiltered() {
 
 				return $this->userIsActor
 					? $this->trans->t('You deleted {sharee}\'s share')
-					: $this->trans->t('{actor} deleted a share');
```

Comment:
Avoiding the sharee here was by intention, for possible privacy concerns.

Reply:
I have restored the text string 😉.